hisprite.s

```diff
@@ -183,7 +190,7 @@ storeAndContinue:
 
 resetX:
 	lda #0
-	bra storeAndContinue
+	beq storeAndContinue
```

Comment:
Note that this sample app has changed a lot recently, so these changes may be moot and/or won't merge well.